equational_theories/FreeMagma.lean

```diff
@@ -21,16 +21,20 @@ notation "Lf" => FreeMagma.Leaf
 
 instance FreeMagma.Magma {α} : Magma (FreeMagma α) := ⟨ Fork ⟩
```

Comment:
I would call it `instance FreeMagma.isMagma` instead.
Then you don't need the `_root_` thing.

Reply:
That's better for sure!